flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveTableSource.java

```diff
@@ -74,12 +77,11 @@
 	private final JobConf jobConf;
 	private final ObjectPath tablePath;
 	private final CatalogTable catalogTable;
+	// Remaining partition specs after partition pruning is performed. Null if pruning is not pushed down.
+	@Nullable
+	private List<Map<String, String>> remainingPartitions = null;
 	private List<HiveTablePartition> allHivePartitions;
```

Comment:
`allHivePartitions` can be a local field.

Reply:
done